### DISCUSSION TITLE
Wiki article with instructions how to install AdNauseam on Chrome is missing.

### DISCUSSION THREAD

**Discussion by yaskovdev** [dhowe/AdNauseam#1903] [General]
Good afternoon!

I cannot find Chrome installation guide anymore. The link https://github.com/dhowe/AdNauseam/wiki/Install-AdNauseam-on-Chrome-or-Chromium-derived-browsers-Without-Google's-Permission leads to a non-existing page.

**Comment by dhowe** [comment]
thanks @yaskovdev link should be fixed now 
points to: https://github.com/dhowe/AdNauseam/wiki/Install-AdNauseam-on-Chromium-based-browsers

**Reply by yaskovdev** [reply]
Please also fix the link on https://adnauseam.io/. Currently it points to https://github.com/dhowe/AdNauseam/wiki/Install-AdNauseam-on-Chrome-Without-Google's-Permission.

**Comment by dhowe** [comment]
done, thanks
